test/build/50_ch-tug.bats

```diff
@@ -234,3 +234,48 @@ EOF
     [[ $output = 'regular file' ]]
     [[ $(cat "${img}/test/source") = 'regular' ]]
 }
+
+@test 'ch-tug image with specified schema' {
+    # Validate that we handle both schema version (1) and (2) image formats.
+    scope standard
+    if ( ! ch-tug --dependencies ); then
+
+        [[ $CH_BUILDER != ch-grow ]]
+        skip "ch-tug missing dependencies"
+    fi
+
+    unpack=$BATS_TMPDIR
+    cache=$unpack/dlcache
+    img=charliecloud%symlink
+
+    export CH_SCHEMA_VER=1
```

Comment:
I don't think you need to export it; I think you can just set it for the `ch-tug` command.